README.md

```diff
@@ -78,14 +78,18 @@ __NOTE__: To roll back to the original driver go to: Device Manager -> Right cli
   * Windows: ```~/Documents/Arduino```
  2. Create the following directories: ```tools/nRF5FlashSoftDevice/tool/```
  3. Download [nRF5FlashSoftDevice.jar](https://github.com/sandeepmistry/arduino-nRF5/releases/download/tools/nRF5FlashSoftDevice.jar) to ```<SKETCHBOOK>/tools/nRF5FlashSoftDevice/tool/```
- 4. Restart the Arduino IDE
- 5. Select your nRF board from the Tools -> Board menu
- 6. Select a SoftDevice from the Tools -> "SoftDevice: " menu
- 7. Select a Programmer (J-Link, ST-Link V2, or CMSIS-DAP) from the Tools -> "Programmer: " menu
- 8. Select Tools -> nRF5 Flash SoftDevice
- 9. Read license agreement
- 10. Click "Accept" to accept license and continue, or "Decline" to decline and abort
- 11. If accepted, SoftDevice binary will be flashed to the board
+ 4. For 64-bit Linux users,  ```libc6:i386```, ```libstdc++6:i386```, ```libncurses5:i386```` and ```libudev1:i386``` need to be installed :
```

Comment:
@tlonch I think it would be better to have these steps in the https://github.com/sandeepmistry/arduino-nRF5#linux section. What do you think?

PS: thanks for opening this PR :)